content/10.diagnostics.md

```diff
@@ -41,7 +40,7 @@ This test shows a sensitivity range of 70-300 copies of the target RNA/ul.
 Furthermore, the assay needs very simple equipment and can be as fast as 30 minutes.
 Finally, the result indicator can be as simple as a color change on a strip [@doi:10.1126/science.abb8400].
 
-##### Limitation of Molecular Tests
+#### 4.1.d Limitation of Molecular Tests
 
 Molecular tests can detect only people that are currently infected, while they cannot detect someone who had developed the infection but has had since recovered.
```

Comment:
```suggestion
Tests that identify SARS-CoV-2 using nucleic-acid-based technologies will identify only individuals with current infections and are not appropriate for identifying individuals who have recovered from a previous infection.
```
Sorry @RLordan, I know the "only" sounds a little odd here compared to how we usually speak, but technically this is where it's supposed to go! (I spent my free time for a while in college copy editing for the school newspaper...who saw it being useful in science!)